pkg/config/p2p.go

```diff
@@ -6,15 +6,12 @@ type P2P struct {
 	Addresses        []string `yaml:"Addresses"`
 	AttemptConnPeers int      `yaml:"AttemptConnPeers"`
 	// BroadcastFactor is the factor (0-100) controlling gossip fan-out number optimization.
-	BroadcastFactor int   `yaml:"BroadcastFactor"`
-	DialTimeout     int64 `yaml:"DialTimeout"`
-	MaxPeers        int   `yaml:"MaxPeers"`
-	MinPeers        int   `yaml:"MinPeers"`
-
-	PingInterval int64 `yaml:"PingInterval"`
-	PingTimeout  int64 `yaml:"PingTimeout"`
-
+	BroadcastFactor    int   `yaml:"BroadcastFactor"`
+	DialTimeout        int64 `yaml:"DialTimeout"`
+	MaxPeers           int   `yaml:"MaxPeers"`
+	MinPeers           int   `yaml:"MinPeers"`
+	PingInterval       int64 `yaml:"PingInterval"`
+	PingTimeout        int64 `yaml:"PingTimeout"`
 	ProtoTickInterval  int64 `yaml:"ProtoTickInterval"`
```

Comment:
Can you make every timing parameter a `time.Duration`? It's a new config section, we can do this.

Reply:
Fixed.